pkgs/tools/networking/kea/default.nix

```diff
@@ -59,7 +63,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     autoreconfHook
     pkg-config
-  ] ++ (with python3.pkgs; [
+  ] ++ (with buildPackages.python3.pkgs; [
```

Comment:
Those should already be spliced

Reply:
I'm not sure why, but they are not. Another example of this is here: https://github.com/NixOS/nixpkgs/blob/c9e3cc43c7baea00c41a4aa24a5e31a08fbfb894/pkgs/misc/uboot/default.nix#L67

Reply:
Try `with python3Packages;` instead.

Reply:
That worked!